internal/backends/winit/event_loop.rs

```diff
@@ -565,6 +565,12 @@ pub fn run(quit_behavior: i_slint_core::backend::EventLoopQuitBehavior) {
     let mut winit_loop = not_running_loop_instance.instance;
     let clipboard = not_running_loop_instance.clipboard;
 
+    // When we need to apply window properties such as title or the width/height (due to constraints),
+    // we receive a custom event. The application implies querying the current window size, so do all of
```

Comment:
I think somethings wrong with the second sentence. I don't understand what you want to say there.
Perhaps we need some sentence about how inner_size only matches the values of the last resize event, so that we need to postpone the apply_window_properties() call.

Reply:
Good point. I've rewritten the paragraph, I hope it's clearer now. I'd like to get it to a state where you can understand it, so I'd be happy to iterate further if needed.